docs/lambda/python-arn-replacement.asciidoc

```diff
@@ -0,0 +1,9 @@
+++++
+<script>
+window.addEventListener("DOMContentLoaded", async () => {
+  addArnGenerator('agent', 'apm-agent-python', 'arn:aws:lambda:${region}:267093732750:layer:elastic-apm-python-${version}');
```

Comment:
This number in the middle of this ARN is wrong, I don't know if we can know this number before we do the first release....

Maybe we can have the robots do a pre-release so we can get this right.